htmresearch/encoders/cio_encoder.py

```diff
@@ -192,10 +194,14 @@ def getUnionEncoding(self, text):
     tokens = TextPreprocess().tokenize(text)
 
     # Count the ON bits represented in the encoded tokens.
-    counts = Counter()
+    counts = SparseMatrix()
```

Comment:
It would be useful to me to have this part as a separate method, something like `getUnionEncodingFromTokens(self, tokens)`.